### PR TITLE
Setup database

### PR DESCRIPTION
**Pull request by Elisha-Tetteyfio**
- Created a database named `animals`.
- Added values for the first few animals.
- Queried some values.

Screenshot 2022-08-23 211336
Screenshot 2022-08-23 211433

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int,
+    name TEXT,
+    date_of_birth date,
+    escape_attempts int,
+    neutered boolean,
+    weight_kg decimal
```
Comment (mateo951) [inline schema.sql:9]: Well done! Animals table added successfully ✔

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES('Agumon', '2020-02-03', 10.23, true, 0);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES('Gabumon', '2018-11-15', 8, true, 2);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES('Pikachu', '2021-01-07', 15.04, false, 1);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES('Devimon', '2017-05-12', 11, true, 5);
```
Comment (mateo951) [inline data.sql:6]: Expected data was correctly introduced 🚀

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name::text LIKE '%mon';
+SELECT name from animals WHERE (date_of_birth < '2019-12-31') AND (date_of_birth > '2016-01-01');
+SELECT name from animals WHERE (neutered = true) AND (escape_attempts < 3);
+SELECT date_of_birth from animals WHERE (name = 'Agumon') OR (name = 'Pikachu');
+SELECT name, escape_attempts from animals WHERE (weight_kg > 10.5);
+SELECT * from animals WHERE (neutered);
+SELECT * from animals WHERE (name != 'Gabumon');
+SELECT * from animals WHERE (weight_kg >= 10.4) AND (weight_kg <=17.3);
```
Comment (mateo951) [inline queries.sql:10]: Fantastic! You answered all the queries with no flaws 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @Elisha-Tetteyfio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (Elisha-Tetteyfio) [reply queries.sql:10]: Thank you @mateo951 
.....with no flaws.... I actually made all my mistakes in a different window 😅

Comment (mateo951) [reply queries.sql:10]: 🤣

**Pull request merged**